### ISSUE TITLE
Accessibility - flash messages do not get announced to screen readers

### ISSUE THREAD

**Issue by metamoni** [joemasilotti/railsdevs.com#438]
**Describe the bug**
Flash messages announce important messages (such as a successful updates to your developer profile) as well as errors the user should be notified of. However, these messages are not announced to screen readers, so people using assistive tech won't be notified of any dynamic content changes. We can fix this with `aria-live` regions, which ensure that any page updates or errors are communicated to assistive tech users.

**To Reproduce**
Steps to reproduce the behavior (using VoiceOver on macOS):
1. Navigate to the developer profile page and click "Edit"
2. Make a change to the developer's profile, but don't save the form yet
3. Enable VoiceOver with `Command + F5`
4. Save the form
5. Notice the "Your profile was updated!" message at the top did not get announced by the screen reader
6. Turn VoiceOver off with `Command + F5`

**Expected behavior**
When performing the same steps as above, VoiceOver should also announce "Your profile was updated!"

**Screenshots**
https://user-images.githubusercontent.com/22390758/169666123-f9e54ba3-6317-4a74-b42b-12e7255c3562.mov

**How to fix this**
Add `aria-live="polite"` to flash messages that appear dynamically. For this PR, that means notices and alerts in `app/views/shared/_flash.html.erb`.

**Additional context**
You can read more about ARIA live regions [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)

**Comment by metamoni** [comment]
I'm working on this 🙂

**Comment by joemasilotti** [comment]
Another accessibility feature I didn't know about. Thank you!
